lib/app.rb

```diff
@@ -15,58 +15,61 @@ def initialize(stdout, stdin)
     @prompter = Prompter.new(stdout)
   end
 
+  def projects
```

Comment:
Nice how you were able to move these into memoized methods! I would move them down to the private section next to the other data sections.

Reply:
✔️